mitosheet/src/mito/pro/taskpanes/ConditionalFormatting/ConditionalFormattingTaskpane.tsx

```diff
@@ -105,14 +111,32 @@ const ConditionalFormattingTaskpane = (props: ConditionalFormattingTaskpaneProps
         }
     )
 
-    const [openFormattingCardIndex, setOpenFormattingCardIndex] = useState(-1)
-
     if (params === undefined) {
         return <DefaultEmptyTaskpane setUIState={props.setUIState}/>
     }
 
     const sheetData = props.sheetDataArray[params.sheet_index];
     const conditionalFormats = params.df_format.conditional_formats
+    // If the user is opening this with a starting column ID, we want to open to that conditional format
+    // If the user is opening this otherwise, we want to open the first conditional format.
+    // And if there are no conditional formats yet, we want to open to -1, which means no conditional format
+    const [openFormattingCardIndex, setOpenFormattingCardIndex] = useState(
+        props.startingColumnIDs ? conditionalFormats.length - 1 : conditionalFormats.length > 0 ? 0 : -1
+    )
+
+    useEffect(() => {
```

Comment:
This is so that it doesn't keep adding new conditional formats every time it renders (maybe I'm missing another way of doing that though?) But I can also add a comment to make it less confusing!